### PR TITLE
:white_check_mark: Unit test some of the backend services

### PR DESCRIPTION
**Pull request by cchampou**
Providing some unit tests for the following services:
- DiscordService
- HttpService
- DatabaseService
- AuthService

### CODE REVIEW

File: backend/src/services/databaseService.ts [hunk 1]
```diff
@@ -1,10 +1,13 @@
 import { connect } from 'mongoose';
+import _ from 'lodash';
```
Comment (Juliotati) [inline backend/src/services/databaseService.ts:3]: 😂lol
Is this good practice, or can it have a better name other than underscore `_`

Comment (cchampou) [reply backend/src/services/databaseService.ts:3]: It's very common, because of `underscore.js`. It's done like this in all project I know 🤷‍♂️

Comment (cchampou) [reply backend/src/services/databaseService.ts:3]: I can otherwise use destructuring

Comment (Juliotati) [reply backend/src/services/databaseService.ts:3]: Man 🤣
It's better to use names with actual words. you can name it `underscore` (if it creates no conflict in the file) at least that will make more sense.

_Pro tip: If everyone does it, it doesn't mean it's "correct"_

**Review by Juliotati — CHANGES_REQUESTED** [review]

**Comment by Juliotati** [comment]
But I see this and get what you mean, but I'm the type that still would NOT do that like it's FUNNY. 
image

File: backend/src/services/databaseService.ts (hunk 1 above)
Comment (cchampou) [reply backend/src/services/databaseService.ts:3]: > Man rofl It's better to use names with actual words. you can name it `underscore` (if it creates no conflict in the file) at least that will make more sense.
> 
> _Pro tip: If everyone does it, it doesn't mean it's "correct"_

What if **[official documentation](https://lodash.com/)** does it? :rofl: 

Then I will use destructuring, I not ready in life to see lodash imported differently :joy:

Comment (cchampou) [reply backend/src/services/databaseService.ts:3]: This is now solved :blush:

Comment (Juliotati) [reply backend/src/services/databaseService.ts:3]: that doc was probably last updated before the language even existed 🤣 , I still would not do it.
For me, the same goes for `var` like, what the IS that ☠️

File: backend/src/services/databaseService.ts [hunk 2]
```diff
@@ -10,4 +10,4 @@ export const init = async () => {
   }
 };
 
-init().then(_.noop).catch(_.noop);
+init().then(noop).catch(noop);
```
Comment (Juliotati) [inline backend/src/services/databaseService.ts:13]: ✨

**Review by Juliotati — APPROVED** [review]

**Pull request merged**